pom.xml

```diff
@@ -26,6 +26,7 @@
   </parent>
   <groupId>org.apache.spark</groupId>
   <artifactId>spark-parent_2.11</artifactId>
+  <!-- When updating version make sure to update ./python/pyspark/version.py as well -->
```

Comment:
Something along the lines of `.splitlines()...trim().startswith('<version>')` would work, and it's easy to error out if it broke, no?